sdks/go/test/run_integration_tests.sh

```diff
@@ -71,21 +130,42 @@ docker images | grep $TAG
 # Push the container
 gcloud docker -- push $CONTAINER
 
-DATAFLOW_WORKER_JAR=$(find ./runners/google-cloud-dataflow-java/worker/build/libs/beam-runners-google-cloud-dataflow-java-fn-api-worker-*.jar)
-echo "Using Dataflow worker jar: $DATAFLOW_WORKER_JAR"
+if [[ "$RUNNER" == "dataflow" ]]; then
+  if [[ -z "$DATAFLOW_WORKER_JAR" ]]; then
+    DATAFLOW_WORKER_JAR=$(find ./runners/google-cloud-dataflow-java/worker/build/libs/beam-runners-google-cloud-dataflow-java-fn-api-worker-*.jar)
+  fi
+  echo "Using Dataflow worker jar: $DATAFLOW_WORKER_JAR"
+elif [[ "$RUNNER" == "flink" ]]; then
+  if [[ -z "$ENDPOINT" ]]; then
+    JOB_PORT=$(python ./sdks/python/apache_beam/utils/ports.py)
```

Comment:
This will be really hard to maintain as python dev will mostly not know about this dependency.
Can we write a small shell script method instead of creating go-python dependency for test.

Reply:
I think I am going to implement the TODO [1] and have the job server process pipe its dynamically chosen port back to the test runner.

[1] https://github.com/apache/beam/blob/7d6b7b89f5832d5d375059f9c1980181b443835b/sdks/python/apache_beam/runners/portability/portable_runner_test.py#L102-L103

Reply:
That was getting too messy, so I opted to just include a simple Python script inline in the interest of not overcomplicating things.